types/generated/components.d.ts

```diff
@@ -161,6 +161,21 @@ export interface SectionsSecondaryBanner extends Schema.Component {
   };
 }
 
+export interface SectionsTimeline extends Schema.Component {
+  collectionName: 'components_sections_timelines';
+  info: {
+    displayName: 'Timeline';
+    description: '';
+  };
+  attributes: {
+    heading: Attribute.Component<'shared.heading'>;
+    Card: Attribute.Component<'shared.timeline-card', true>;
```

Comment:
Suggestion:
Let's have consistency in properties case. `Card` => `card`

Reply:
Renamed